pkgs/by-name/pu/pushup/package.nix

```diff
@@ -0,0 +1,35 @@
+{ fetchFromGitHub, buildGoModule, lib, go, makeWrapper }:
+buildGoModule rec {
```

Comment:
```suggestion
{ fetchFromGitHub, buildGoModule, lib, go, makeWrapper }:

buildGoModule rec {
```

Reply:
this is a formatting nit and for me it's fine either ways, we will have a official for-matter soon :tm: